Vokal-Cocoa Touch Application Base.xctemplate/fastlane/Fastfile

```diff
@@ -277,7 +277,7 @@ platform :ios do
       create_keychain(
         name: keychain_name,
         password: keychain_password,
-        default_keychain: false,
+        default_keychain: true,
```

Comment:
[I think the description of this parameter says](https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/actions/create_keychain.rb#L78-L81) "Should the newly created Keychain be the new system default keychain", so while I doubt it matters for CI, it might be bad locally?

Reply:
For CI, it only seems to work as `true` for me. Locally, this should not matter is we are within a `is_ci` check here.

Reply:
Ah, ok.